build.sbt

```diff
@@ -443,6 +443,25 @@ lazy val federation = project
     )
   )
 
+lazy val docs = project
+  .in(file("mdoc"))
+  .enablePlugins(MdocPlugin)
+  .settings(commonSettings)
+  .settings(
+    scalaVersion       := scala213,
+    crossScalaVersions := Seq(scala213),
+    name               := "caliban-docs",
+    mdocIn             := (ThisBuild / baseDirectory).value / "vuepress" / "docs",
+    run / fork         := true,
+    scalacOptions -= "-Xfatal-warnings",
```

Comment:
There is only one warning, actually:

```
warning: middleware.md:133:55: The outer reference in this type test cannot be checked at run time.
  case err @ ExecutionError(_, _, _, Some(exampleError: ExampleAppEncodableError), _) =>
```

But I cannot find a simple way to sort it out.

Reply:
Yeah I saw it too. That's good enough 🤷‍♂️